webapp/src/pages/EditAvatarPage/EditAvatarPage.tsx

```diff
@@ -40,39 +51,40 @@ export default function EditAvatarPage() {
     imgFile: null,
     imgSrc: '',
   });
-  const getCurrentUser = useCallback(
-    () => usersApi.userControllerGetCurrentUser(),
-    // eslint-disable-next-line
-    [imgData],
-  );
-  const { data: user } = useData(getCurrentUser);
-  const navigate = useNavigate();
   const reverseTransform = useCallback(
     (value: number) => -value * EDITABLE_AVATAR_SCALE,
     [],
   );
   const { picturePosition, handleDown, handleMove, handleUp } = useDrag({
-    startingPosition: user ? { x: user.avatarX, y: user.avatarY } : null,
+    startingPosition: authUser
+      ? { x: authUser.avatarX, y: authUser.avatarY }
+      : null,
     reverseTransform,
   });
 
   const { imgName, imgFile } = imgData;
   const FormatNumber = (value: number) =>
     Math.round(value * EDITABLE_AVATAR_SCALE_REVERSE);
+  const avatarX = FormatNumber(picturePosition.x);
+  const avatarY = FormatNumber(picturePosition.y);
   const submitPlacement = async () => {
     usersApi
       .userControllerUpdateCurrentUserRaw({
         updateUserDto: {
-          avatarX: FormatNumber(picturePosition.x),
-          avatarY: FormatNumber(picturePosition.y),
+          avatarX,
+          avatarY,
         },
       })
-      .then(() =>
+      .then(() => {
         setStatus({
           type: 'success',
           message: 'Image visible area saved correctly.',
-        }),
-      )
+        });
+        setAuthUser((prevState) => {
+          if (!prevState) return null;
+          return { ...prevState, avatarX, avatarY };
+        });
```

Comment:
Since we are using persited authUser, we should update its state everytime we update it on backend

Reply:
praise: I like this change. Thank you!